.extlib/simplesamlphp/modules/saml/src/Auth/Source/SP.php

```diff
@@ -629,16 +648,22 @@ private function startSSO2(Configuration $idpMetadata, array $state): void
             $dst = $idpMetadata->getEndpointPrioritizedByBinding(
                 'SingleSignOnService',
                 [
-                    Constants::BINDING_HTTP_ARTIFACT,
                     Constants::BINDING_HTTP_REDIRECT,
                     Constants::BINDING_HTTP_POST,
+                    Constants::BINDING_HTTP_ARTIFACT,
```

Comment:
Additionally, can we get a modification comment above this block mentioning the reordering to put artifact last to maintain existing behaviour.